app/uk/gov/hmrc/fhregistrationfrontend/views/businessPartners/v1/business_partners_corporate_body.scala.html

```diff
@@ -0,0 +1,206 @@
+@*
```

Comment:
Should the folder be `/businessPartners` rather than `/businessPartners/v1`?

Not sure if that's FHDDS code nomenclature or WIP naming?

Reply:
well there's already a v2...

Reply:
this was to be sympathetic to that structure - it's the same type of templates as v2 already holds (for the rewrite)